src/reversion/models.py

```diff
@@ -214,3 +194,6 @@ def __str__(self):
     #Meta
     class Meta:
         app_label = 'reversion'
+        index_together = [
+            ["content_type", "object_id"],
```

Comment:
I think this needs to be ["object_id", "content_type"], since that'll mean the index can be used just for "object_id", for queries that don't need the content_type.